test/apis/v1alpha1/api_validation_integration_test.go

```diff
@@ -0,0 +1,907 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1alpha1
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+
+	v1alpha1 "go.goms.io/fleet-networking/api/v1alpha1"
```

Comment:
```suggestion
	fleetnetv1alpha1 "go.goms.io/fleet-networking/api/v1alpha1"
```
to keep it consistent with others for a better readability.